src/CommunityToolkit.Maui.Core/Essentials/FileSaver/FileSaverImplementation.android.cs

```diff
@@ -20,7 +20,7 @@ static async Task<string> InternalSaveAsync(string initialPath, string fileName,
 	{
 		if (!OperatingSystem.IsAndroidVersionAtLeast(26) && !string.IsNullOrEmpty(initialPath))
 		{
-			Trace.WriteLine("Specifying an initial path is only supported on Android 26 and later.");
+			Trace.TraceError("Specifying an initial path is only supported on Android 26 and later.");
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.